pkg/formats/elasticsearch/elasticsearch.go

```diff
@@ -140,6 +139,30 @@ func (f *ElasticsearchFormat) Rollup(rolls []rollup.Rollup) (*kt.Output, error)
 	return kt.NewOutputWithProvider([]byte(strings.Join(esBulkData, "")), rolls[0].Provider, kt.RollupOutput), nil
 }
 
+func (f *ElasticsearchFormat) handleSynth(in *kt.JCHF) map[string]interface{} {
+	metrics := util.GetSynMetricNameSet(in.CustomInt["result_type"])
+	attr := in.Flatten()
+	strip(attr)
+
+	for m, name := range metrics {
+		if _, ok := attr[m]; ok {
+			attr[name.Name] = attr[m]
+			delete(attr, m)
+		}
+	}
+
+	// If there's a traceroute, try to unserialize this one.
```

Comment:
Should we move these into a helper package or something where other exporters could use or is this specific to this one?  Doesn't have to be now just a thought.

Reply:
Good idea. At some point I want to go back and make all the formatters just about format, not this kind of data enrichment. But for now I'll just ask the user what they think.